test/modules/TestResultCoordinator/Settings.cs

```diff
@@ -150,11 +163,9 @@ static Settings Create()
 
         public bool OptimizeForPerformance { get; }
 
-        public TimeSpan TestDuration { get; }
-
         public TimeSpan TestStartDelay { get; }
 
-        public TimeSpan DurationBeforeVerification { get; }
+        public TimeSpan SendReportFrequency { get; }
```

Comment:
Although after looking at the usage in the case of connectivity, it currently only kicks in after the delayBeforeWork which is when the test finishes.

Reply:
I was originally thinking that it could be useful for connectivity tests, but honestly I don't think it is. If it is ever is necessary in the future we can add it then.

Reply:
Modified to be longhaul specific setting